File: app/src/main/java/com/vn/ezlearn/modelresult/QuestionResult.java
package com.vn.ezlearn.modelresult;

import com.vn.ezlearn.models.Content;

import java.util.List;

/**
 * Created by FRAMGIA\nguyen.duc.manh on 10/10/2017.
 */

public class QuestionResult {
    public Boolean success;
    public QuestionData data;

    public static class QuestionData {
        public List<Content> data;
    }
}

File: app/src/main/java/com/vn/ezlearn/modelresult/QuestionResult.kt
package com.vn.ezlearn.modelresult

import com.vn.ezlearn.models.Question

/**
 * Created by FRAMGIA\nguyen.duc.manh on 10/10/2017.
 */

class QuestionResult {
    var success: Boolean = false
    var data: QuestionData? = null

    class QuestionData {
        var data: List<Question>? = null
        var message: String? = null
    }
}
